test/unit/controllers/AuthControllerTests.ts

```diff
@@ -0,0 +1,89 @@
+import { describe, it } from "node:test";
+import * as AuthController from "../../../src/controllers/AuthController";
+import * as AuthService from "../../../src/services/AuthService";
+import { expect } from 'chai';
+
+import sinon from 'sinon';
+
+const JWTTOKEN = "eyJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE3MjE3NjM3"
++ "MTQsImV4cCI6MTcyMTc5MjUxNCwiUm9sZSI6MSwic3ViIjoiYWRtaW4iLCJ"
++ "pc3MiOiJ0ZWFtMS1hcGkifQ.13PjVdPseFyBE8AQrjHSSM0Spx-1tkYnwHjR5IVITeU";
+
+declare module "express-session" {
+    interface SessionData {
+      token: string;
+    }
+  }
+
+describe('LoginController', function () {
+    //Tests if login for appears
+    describe('getLoginForm', function () {
+        after (() => {
+            sinon.reset();
+        });
+        it('should render the login form', async () => {
+
+            const stub = sinon.stub(AuthService, 'getToken').resolves(JWTTOKEN);
+
+            const req = { session: { token: "" } };
+            const res = { render: sinon.spy() };
+
+            //eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
remove?

Reply:
its needed so that linter doesnt fail the test. That's how we were showed to do the tests in the academy but we didnt use linter